cocos/2d/CCActionInterval.cpp

```diff
@@ -119,7 +119,7 @@ void ActionInterval::step(float dt)
     if (_firstTick)
     {
         _firstTick = false;
-        _elapsed = 0;
+        _elapsed = MATH_EPSILON;
```

Comment:
Could you explain this fix? What is wrong with 0?

Reply:
PR https://github.com/cocos2d/cocos2d-x/pull/19158 result 
cpp-tests/3.Action tests (54,55)

![-3-1](https://user-images.githubusercontent.com/40414978/49347704-9ef61c00-f6db-11e8-9a71-f2a1ea5ed86c.png)

test cause failure.